src/api/error.rs

```diff
@@ -42,4 +42,7 @@ pub enum Error {
 
     #[error("grpcio conn failed: {0}")]
     GrpcioJoin(#[from] grpcio::Error),
+
+    #[error("Wrong grpc address: {0}")]
+    WrongGrpcAddress(String),
```

Comment:
Perhaps `WrongServerAddr` is more consistent with the definition, and user set up `server_addr` must be the `ip:port` specification.

Reply:
@CherishCai It is renamed.